lcoe_calculator/base_processor.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         @param data_workbook_fname - name of workbook
         @param case - financial case to run: 'Market' or 'R&D'
         @param crp - capital recovery period: 20, 30, or 'TechLife'
+        @param tcc - tax credit case: 'ITC only' or 'PV PTC and Battery ITC' Only required for the PV plus battery technology.
```

Comment:
❤️

Reply:
Random question, what happens if someone sets the tcc for LBW or Nuclear?

Reply:
It would get passed through to the extractor and set in Excel, but wouldn't change the numbers at all.

Reply:
I'm fine with that if you are. I don't think  the additional complexity of preventing a user from using it incorrectly is worth it minor benefit.